
Log Entry : 
12/02/2016 08:48:34
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at System.Web.HttpResponse.End()
   at System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   at System.Web.HttpResponse.Redirect(String url)
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 171
mscorlib
__________________________

Log Entry : 
12/02/2016 08:49:15
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at System.Web.HttpResponse.End()
   at System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   at System.Web.HttpResponse.Redirect(String url)
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 176
mscorlib
__________________________

Log Entry : 
12/02/2016 08:49:15
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 178
App_Web_login.aspx.cdcab7d2.igdiystg
__________________________

Log Entry : 
12/02/2016 08:49:22
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at System.Web.HttpResponse.End()
   at System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   at System.Web.HttpResponse.Redirect(String url)
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 176
mscorlib
__________________________

Log Entry : 
12/02/2016 08:49:22
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 178
App_Web_login.aspx.cdcab7d2.igdiystg
__________________________

Log Entry : 
12/02/2016 08:49:33
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at System.Web.HttpResponse.End()
   at System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   at System.Web.HttpResponse.Redirect(String url)
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 176
mscorlib
__________________________

Log Entry : 
12/02/2016 08:49:33
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 178
App_Web_login.aspx.cdcab7d2.igdiystg
__________________________

Log Entry : 
12/02/2016 08:50:31
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at System.Web.HttpResponse.End()
   at System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   at System.Web.HttpResponse.Redirect(String url)
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 176
mscorlib
__________________________

Log Entry : 
12/02/2016 08:50:31
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 178
App_Web_login.aspx.cdcab7d2.igdiystg
__________________________

Log Entry : 
12/02/2016 08:50:42
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at System.Web.HttpResponse.End()
   at System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   at System.Web.HttpResponse.Redirect(String url)
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 176
mscorlib
__________________________

Log Entry : 
12/02/2016 08:50:42
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 178
App_Web_login.aspx.cdcab7d2.igdiystg
__________________________

Log Entry : 
12/02/2016 08:52:51
Error in: http://localhost:58911/login.aspx. Error Message:System.InvalidCastException
Specified cast is not valid.
   at System.Data.SqlClient.SqlBuffer.get_Byte()
   at System.Data.SqlClient.SqlDataReader.GetByte(Int32 i)
   at Pronto.ERP.Dal.ProveedorDB.FillDataRecord(IDataRecord myDataRecord)
   at Pronto.ERP.Dal.ProveedorDB.GetList(String SC)
   at Pronto.ERP.Bll.ProveedorManager.GetList(String SC) in C:\Users\Administrador\Documents\bdl\pronto\BussinessLogic\ProveedorManager.vb:line 12
   at Pronto.ERP.Bll.BDLMasterEmpresasManagerMigrar.AddEmpresaToSession(String IdEmpresa, Object& Session, String sConexBDLMaster, Object Yo) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\App_Code\vbcode\ManagerDebug\BDLMasterManagerAppCode.vb:line 339
System.Data
__________________________

Log Entry : 
12/02/2016 08:53:25
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at System.Threading.Thread.AbortInternal()
   at System.Threading.Thread.Abort(Object stateInfo)
   at System.Web.HttpResponse.AbortCurrentThread()
   at System.Web.HttpResponse.End()
   at System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   at System.Web.HttpResponse.Redirect(String url)
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 176
mscorlib
__________________________

Log Entry : 
12/02/2016 08:53:25
Error in: http://localhost:58911/login.aspx. Error Message:System.Threading.ThreadAbortException
Thread was being aborted.
   at Login.Page_Load(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\Login.aspx.vb:line 178
App_Web_login.aspx.cdcab7d2.igdiystg
__________________________

Log Entry : 
12/02/2016 08:53:56
Error in: http://localhost:58911/login.aspx. Error Message:System.InvalidCastException
Specified cast is not valid.
   at System.Data.SqlClient.SqlBuffer.get_Byte()
   at System.Data.SqlClient.SqlDataReader.GetByte(Int32 i)
   at Pronto.ERP.Dal.ProveedorDB.FillDataRecord(IDataRecord myDataRecord)
   at Pronto.ERP.Dal.ProveedorDB.GetList(String SC)
   at Pronto.ERP.Bll.ProveedorManager.GetList(String SC) in C:\Users\Administrador\Documents\bdl\pronto\BussinessLogic\ProveedorManager.vb:line 12
   at Pronto.ERP.Bll.BDLMasterEmpresasManagerMigrar.AddEmpresaToSession(String IdEmpresa, Object& Session, String sConexBDLMaster, Object Yo) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\App_Code\vbcode\ManagerDebug\BDLMasterManagerAppCode.vb:line 339
System.Data
__________________________

Log Entry : 
12/02/2016 08:54:02
Error in: http://localhost:58911/ProntoWeb/CartasDePortes.aspx?tipo=Todas. Error Message:System.NullReferenceException
Object reference not set to an instance of an object.
   at CartasDePortes.BindTypeDropDown() in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\ProntoWeb\CartasDePortes.aspx.vb:line 85
App_Web_cartasdeportes.aspx.4199bbd4.t2n6ygus
__________________________

Log Entry : 
12/02/2016 08:54:13
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientes.aspx. Error Message:System.NullReferenceException
Object reference not set to an instance of an object.
   at CartaDePorteInformesAccesoClientes.BindTypeDropDown() in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\ProntoWeb\CartaDePorteInformesAccesoClientes.aspx.vb:line 227
App_Web_cartadeporteinformesaccesoclientes.aspx.4199bbd4._bcvpgg9
__________________________

Log Entry : 
12/02/2016 08:56:15
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:BLDCORREDOR 01/01/1753
__________________________

Log Entry : 
12/02/2016 08:56:15
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:BLDCORREDOR 01/01/2100
__________________________

Log Entry : 
12/02/2016 08:56:43
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:  Informe: 
__________________________

Log Entry : 
12/02/2016 08:59:55
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:BLDCORREDOR 01/01/1753
__________________________

Log Entry : 
12/02/2016 08:59:55
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:BLDCORREDOR 01/01/2100
__________________________

Log Entry : 
12/02/2016 09:19:17
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:System.Data.Entity.Core.EntityException
The underlying provider failed on Open.
   at System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   at System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass3.<GetResults>b__1()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   at System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   at System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at CartaDePorteInformesAccesoClientesBLDcorredor.AsignaInformeAlReportViewer(Boolean bDescargaExcel) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\ProntoWeb\CartaDePorteInformesAccesoClientesBLDcorredor.aspx.vb:line 505
   at CartaDePorteInformesAccesoClientesBLDcorredor.btnRefrescar_Click(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\ProntoWeb\CartaDePorteInformesAccesoClientesBLDcorredor.aspx.vb:line 142
   at System.Web.UI.WebControls.Button.OnClick(EventArgs e)
   at System.Web.UI.WebControls.Button.RaisePostBackEvent(String eventArgument)
   at System.Web.UI.WebControls.Button.System.Web.UI.IPostBackEventHandler.RaisePostBackEvent(String eventArgument)
   at System.Web.UI.Page.RaisePostBackEvent(IPostBackEventHandler sourceControl, String eventArgument)
   at System.Web.UI.Page.RaisePostBackEvent(NameValueCollection postData)
   at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
EntityFramework
__________________________

Log Entry : 
12/02/2016 09:19:26
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message: -  Error en ExecDinamico. - System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Tiempo de espera de la operación de espera agotado.) ---> System.ComponentModel.Win32Exception (0x80004005): Tiempo de espera de la operación de espera agotado
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Pronto.ERP.Dal.GeneralDB.ExecDinamico(String SC, String comandoSQLdinamico, Int32 timeoutSegundos)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:258,State:0,Class:20 - EXEC sp_who2 8 0
__________________________

Log Entry : 
12/02/2016 09:19:26
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Tiempo de espera de la operación de espera agotado.)
__________________________

Log Entry : 
12/02/2016 09:19:27
Error in: http://localhost:58911/ProntoWeb/CartaDePorteInformesAccesoClientesBLDcorredor.aspx. Error Message:System.Data.SqlClient.SqlException
A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Tiempo de espera de la operación de espera agotado.)
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Pronto.ERP.Dal.GeneralDB.ExecDinamico(String SC, String comandoSQLdinamico, Int32 timeoutSegundos)
   at Pronto.ERP.Bll.EntidadManager.ExecDinamico(String SC, String sComandoDinamico, Int32 timeoutSegundos) in C:\Users\Administrador\Documents\bdl\pronto\BussinessLogic\EntidadManager.vb:line 803
   at CartaDePorteManager.Log_sp_who2(String sc) in C:\Users\Administrador\Documents\bdl\pronto\BussinessLogic\ManagerDebug\CartaDePorteManager.vb:line 7898
   at ASP.global_asax.Application_Error(Object sender, EventArgs e) in C:\Users\Administrador\Documents\bdl\pronto\prontoweb\global.asax:line 433
.Net SqlClient Data Provider
__________________________

Log Entry : 
12/02/2016 14:47:39
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_144739_MMC.xls  Informe: Descargas,  del 01/11/2016 al 02/12/2016           (Exporta: Entregas, Punto de venta: Todos, Criterio: ALGUNOS)
__________________________

Log Entry : 
12/02/2016 14:51:48
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Por generar el archivo C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145143_CVD.xls
__________________________

Log Entry : 
12/02/2016 14:51:48
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Archivo generado C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145143_CVD.xls
__________________________

Log Entry : 
12/02/2016 14:51:51
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Por generar el archivo C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145149_IWX.xls
__________________________

Log Entry : 
12/02/2016 14:51:51
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Archivo generado C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145149_IWX.xls
__________________________

Log Entry : 
12/02/2016 14:51:54
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Nombre  'Listado general de Cartas de Po$'
__________________________

Log Entry : 
12/02/2016 14:51:54
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Tablas  1
__________________________

Log Entry : 
12/02/2016 14:51:54
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Convertido C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145149_IWX.xls Lineas: 10
__________________________

Log Entry : 
12/02/2016 14:52:10
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Por generar el archivo C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145208_XXQ.xls
__________________________

Log Entry : 
12/02/2016 14:52:10
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Archivo generado C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145208_XXQ.xls
__________________________

Log Entry : 
12/02/2016 14:52:14
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Por generar el archivo C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145211_MOO.xls
__________________________

Log Entry : 
12/02/2016 14:52:14
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Archivo generado C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145211_MOO.xls
__________________________

Log Entry : 
12/02/2016 14:52:14
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Nombre  'Listado general de Cartas de Po$'
__________________________

Log Entry : 
12/02/2016 14:52:14
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Tablas  1
__________________________

Log Entry : 
12/02/2016 14:52:14
Error in: http://localhost:58911/ProntoWeb/CDPMailing.aspx. Error Message:Convertido C:\Users\Administrador\AppData\Local\Temp\Listado general 02dic2016_145211_MOO.xls Lineas: 10
__________________________
